modules/sporestrofit.py

```diff
@@ -0,0 +1,113 @@
+import json
+import logging
+from zoneinfo import available_timezones
+import requests
+from dataclasses import dataclass
+import urllib
+logging.basicConfig(level=logging.INFO)
+
+@dataclass
+class Sporestrofit:
+    host: str = "http://app.sporetrofit.com:8080"
+    headers = {
+        'Accept': "application/json",
+        'Content-Type': "application/x-www-form-urlencoded",
+        'User-Agent': "tp-stage/2.0.10 (com.fit-foxconn.TPSportsCenter; build:1; iOS 15.5.0) Alamofire/2.0.10"
+    }
+    temp_ID: str = "81479C1C-B830-4941-872E-7F0D5216F090"
+    UPID:str = "d6f4b8d3-b581-4cc2-8976-da98a9f43c73"
+    COID:str = "TP"
+    category_ID:str = "Badminton"
+    UUID:str = "81479C1C-B830-4941-872E-7F0D5216F090"
```

Comment:
Question: Do the name `UPID`, `COID`, and `UUID` are valid to Python variable naming rules? Or should named `upid`, `coid`, and `uuid`?

Reply:
I think if we move them to .env it should be `UPID`,`COID` because it is a global variable.